### DISCUSSION TITLE
There is absolutely no documentation on how to load in $d and $n formats

### DISCUSSION THREAD

**Discussion by geewhizbang** [intlify/bundle-tools#96] [Q&A]
I would like to use the $d and $n formats in i18n, but there is absolutely no documentation anywhere on how to use them with this plugin. It doesn't work to add formats to messages, they are apparently added separately somehow in some utterly undefined way.

Where do I do this? All the examples for i18n tend to install i18n from a cdn on one page, and don't give any examples on how this is done globally with a plugin like this.

You would think that it needs to be done here in i18n.ts, but I don't seem to have a property in createI18n that allows them to be passed in:

    const install = (app: App) => {
        const i18n = createI18n({
           legacy: false,
           locale,
          globalInjection: true,
          messages,
          fallbackLocale: 'en-US'
        })
       app.use(i18n)
  }

**Comment by kazupon** [comment]
That doc is here:

- datetime formatting
  - https://vue-i18n.intlify.dev/guide/essentials/datetime.html
- number formatting
  - https://vue-i18n.intlify.dev/guide/essentials/number.html
